app/src/main/res/drawable/bottom_left_rounded_rect_coming_soon_background.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android">
+  <corners android:bottomRightRadius="0dp"
+    android:bottomLeftRadius="12dp"
+    android:topLeftRadius="0dp"
+    android:topRightRadius="4dp"/>
+  <gradient
+    android:angle="360"
+    android:startColor="#7659B6"
+    android:endColor="#674172"
+    android:type="linear" />
+</shape>
```

Comment:
Ditto for this file, why is this file needed in this PR?

Reply:
Fixed.